runtime/drivers/duckdb/model_manager.go

```diff
@@ -25,6 +25,7 @@ type ModelOutputProperties struct {
 	Table               string                      `mapstructure:"table"`
 	Materialize         *bool                       `mapstructure:"materialize"`
 	UniqueKey           []string                    `mapstructure:"unique_key"`
+	ReplaceKey          []string                    `mapstructure:"replace_key"`
```

Comment:
Don't think we need replace strategy if both merge and replace works the same way. Also key should be same (`unique_key`).